jraph/_src/models.py

```diff
@@ -52,17 +52,22 @@
 # (edge features, sender node features, receiver node features, globals) ->
 # attention weights
 AttentionLogitFn = Callable[
-    [EdgeFeatures, SenderFeatures, ReceiverFeatures, Globals], ArrayTree]
+  [EdgeFeatures, SenderFeatures, ReceiverFeatures, Globals], ArrayTree]
 
 # Signature:
 # (edge features, weights) -> edge features for node update
 AttentionReduceFn = Callable[[EdgeFeatures, ArrayTree], EdgeFeatures]
 
+# Signature:
+# (edges to be normalized, segment ids, number of segments) ->
+# normalized edges
+AttentionNormalizeFn = Callable[[EdgeFeatures, jnp.ndarray, int], Globals]
+
 # Signature:
 # (edge features, sender node features, receiver node features, globals) ->
 # updated edge features
 GNUpdateEdgeFn = Callable[
```

Comment:
No need to change this.

Reply:
You want me to remove the AttentionNormalizeFn type hint?

Reply:
I thought I had added this to the other formatting fix. This looks great, thanks :)